src/components/PeopleManagement/index.jsx

```diff
@@ -66,6 +75,14 @@ const PeopleManagementPage = ({ enterpriseId }) => {
     }
   }
 
+  const handleOnClickCreateGroup = () => {
+    openModal();
+    sendEnterpriseTrackEvent(
```

Comment:
nit: Code coverage warning here, seems we do not have tests that open Create Group modal from People Management page.